sdk/inc/azure/core/az_context.h

```diff
@@ -33,6 +33,7 @@ typedef struct az_context az_context;
  *
  * @details The root node in the tree (ultimate parent).
  */
+
 struct az_context
 {
   struct
```

Comment:
I think that the comment should go above the `struct` in line 39.

Reply:
I have made the changes and now the the comment can be seen in the doxygen html under `Note`. As for the clang format My ubuntu version is 22.04 and this command does not seem to work. I installed clang, but clang format-9 gives me error same as previous one.

`find . \( -iname '*.h' -o -iname '*.c' \) -exec clang-format-9 -i {} \;`

Reply:
It's unclear if you've installed version 9 of clang-format. "clang-format" might be a different version and won't format the code the same way.

Did this work? 

`sudo apt install clang-format-9`

Afterwards, test using

```sh
clang-format-9 --version
```

Reply:
> It's unclear if you've installed version 9 of clang-format. "clang-format" might be a different version and won't format the code the same way.
> 
> Did this work?
> 
> `sudo apt install clang-format-9`

 Nope, it says `Unable to locate package clang-format-9`